docs/en/contributor/Contributor-Tools.md

```diff
@@ -227,26 +212,17 @@ $ mvn clean install -Pgenerate
 ```
 
 ## Usage of `./bin/alluxio`
+ 
+Please refer to [Alluxio commands]({{ '/en/operation/User-CLI.html' | relativize_url }})
+for all available commands.
+
+Some commands have different prerequisites.
 
-Most commands in `bin/alluxio` are for developers. The following table explains the description and
-the syntax of each command.
-
-<table class="table table-striped">
-    <tr><th>Command</th><th>Args</th><th>Description</th></tr>
-    {% for dscp in site.data.table.developer-tips %}
-        <tr>
-            <td>{{dscp.command}}</td>
-            <td>{{dscp.args}}</td>
-            <td>{{site.data.table.en.developer-tips[dscp.command]}}</td>
-        </tr>
-    {% endfor %}
-</table>
-
-In addition, these commands have different prerequisites. The prerequisite for the `format`,
-`formatWorker`, `journalCrashTest`, `readJournal`, `runClass`, `version`, `validateConf` and `validateEnv` commands is
-that you have already built Alluxio (see
-[Build Alluxio Master Branch]({{ '/en/contributor/Building-Alluxio-From-Source.html' | relativize_url }}) about how to
-build Alluxio manually).
-Further, the prerequisite for the `fs`, `logLevel`, `runTest` and `runTests` commands is that
-the Alluxio service is up and running.
```

Comment:
We no longer need the table here, as it's been outdated for a while. @Xenorith do you think we still need the csv resources?

Reply:
I think the User-CLI page will do just fine listing all the commands in much more details.

Reply:
I think previously we want to separate commands under `bin/alluxio` to be 
1) using by users -> put in User-CLI page (like format, fs, fsadmin, getConf...)
2) using by developers -> put in this page (like runUfsTests, confDocGen)

Reply:
but yeah, maybe there are better ways to present those commands and descriptions

Reply:
Developer operations section sounds good to me

Reply:
On `developer operations`, I think the only one that's really `developer tool` per se is the `runClass`. Other ones may be better used and understood if you can read the code, but not `developer-only` to me. 

I prefer to make the difference "when you run this command whether the Alluxio code has to be built and whether the cluster has to be up". This can go to the User-CLI page, as this particular page is `Contributor Tools` after all.

Reply:
i'm fine with omitting the duplicate explanation here and making the improvements in the user cli page as a later PR. important thing is to get rid of out of date table in the contribute tools page